### PR TITLE
Updated docs formatting

### PR DESCRIPTION
**Pull request by neighlyd**
tf.feature_column.categorical_column_with_vocabulary_file defined in python/feature_column/feature_column_v2.py.

### CODE REVIEW

**Comment by rthadur** [comment]
@neighlyd thanks for your contribution, please sign CLA

**Review by dynamicwebpaige — DISMISSED** [review]
LGTM! Thanks for these fixes.

**Review by mihaimaruseac — APPROVED** [review]

**Comment by rthadur** [comment]
@neighlyd gentle ping to sign CLA

**Comment by neighlyd** [comment]
I signed it this morning!

**Comment by neighlyd** [comment]
I apologize, it looks like I hadn't replied to the googlebot's CLA message! I went ahead and did that.

File: tensorflow/python/feature_column/feature_column_v2.py [hunk 1]
```diff
@@ -1604,8 +1604,8 @@ def categorical_column_with_vocabulary_file_v2(key,
   ```
 
   Example with `default_value`:
-  File '/us/states.txt' contains 51 lines - the first line is 'XX', and the
-  other 50 each have a 2-character U.S. state abbreviation. Both a literal 'XX'
+  File `'/us/states.txt'` contains 51 lines - the first line is `'XX'`, and the
+  other 50 each have a 2-character U.S. state abbreviation. Both a literal `'XX'`
```
Comment (mihaimaruseac) [inline tensorflow/python/feature_column/feature_column_v2.py]: Linter reports error here

```
FAIL: Found 1 non-whitelited pylint errors:
tensorflow/python/feature_column/feature_column_v2.py:1608: [C0301(line-too-long), ] Line too long (81/80)
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]
Thank you

**Pull request merged**